{"time":"2025-01-29T22:24:33.2727863+01:00","level":"INFO","msg":"using version","core version":"0.18.7"}
{"time":"2025-01-29T22:24:33.2727863+01:00","level":"INFO","msg":"created symlink","path":"C:\\Users\\BHARGAV BADE\\Multivar LSTM\\wandb\\run-20250129_222433-efbsbalz\\logs\\debug-core.log"}
{"time":"2025-01-29T22:24:33.6292413+01:00","level":"INFO","msg":"created new stream","id":"efbsbalz"}
{"time":"2025-01-29T22:24:33.6292413+01:00","level":"INFO","msg":"stream: started","id":"efbsbalz"}
{"time":"2025-01-29T22:24:33.6292413+01:00","level":"INFO","msg":"writer: Do: started","stream_id":"efbsbalz"}
{"time":"2025-01-29T22:24:33.6292413+01:00","level":"INFO","msg":"sender: started","stream_id":"efbsbalz"}
{"time":"2025-01-29T22:24:33.6292413+01:00","level":"INFO","msg":"handler: started","stream_id":"efbsbalz"}
{"time":"2025-01-29T22:24:34.2362833+01:00","level":"INFO","msg":"Starting system monitor"}
{"time":"2025-01-29T23:38:16.9132072+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:38:26.5708556+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.175:54826->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-01-29T23:38:31.3816329+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:38:31.3816329+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:38:47.5594364+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:38:47.5615826+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:39:08.6025997+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:39:08.6025997+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:41:55.4016068+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:42:09.5207717+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:42:11.9990093+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.175:55019->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-01-29T23:42:26.1301751+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:42:26.1303244+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:42:42.3891972+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:42:42.3897093+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:43:03.9372338+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:43:03.9391406+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-29T23:43:32.8495006+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-30T00:09:19.7170938+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-30T00:09:28.4191639+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.175:55079->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-01-30T00:09:33.9841157+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-01-30T00:09:33.9841157+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/bhargav-bade/Multivar_TS_LSTM_Forecasting/efbsbalz/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
